packages/roosterjs-editor-plugins/lib/pluginUtils/DragAndDropHelper.ts

```diff
@@ -6,31 +6,47 @@ import DragAndDropHandler from './DragAndDropHandler';
  * @internal
  * Compatible mouse event names for different platform
  */
-interface MouseEventNames {
+interface MouseEventInfo {
     MOUSEDOWN: string;
     MOUSEMOVE: string;
     MOUSEUP: string;
+    getPageXY: (e: Event) => number[];
 }
 
 /**
- * Generate event names based on different platforms to be compatible with desktop and mobile browsers
+ * Generate event names and getXY function based on different platforms to be compatible with desktop and mobile browsers
  */
-const MOUSE_EVENT_NAMES: MouseEventNames = (() => {
+const MOUSE_EVENT_INFO: MouseEventInfo = (() => {
     if (Browser.isMobileOrTablet) {
         return {
             MOUSEDOWN: 'touchstart',
             MOUSEMOVE: 'touchmove',
             MOUSEUP: 'touchend',
+            getPageXY: getTouchEventPageXY,
         }
     } else {
         return {
             MOUSEDOWN: 'mousedown',
             MOUSEMOVE: 'mousemove',
             MOUSEUP: 'mouseup',
+            getPageXY: getMouseEventPageXY,
         }
     }
 })()
 
+function getMouseEventPageXY(e: MouseEvent): number[] {
+    return new Array(e.pageX, e.pageY);
```

Comment:
you can just do 
return [e.pageX, e.pageY]

Reply:
Updated! Thanks